core/src/main/java/com/datastax/oss/driver/api/core/session/SessionBuilder.java

```diff
@@ -617,7 +638,13 @@ public SessionT build() {
   @NonNull
   protected final CompletionStage<CqlSession> buildDefaultSessionAsync() {
     try {
-      DriverConfigLoader configLoader = buildIfNull(this.configLoader, this::defaultConfigLoader);
+
+      ProgrammaticArguments programmaticArguments = programmaticArgumentsBuilder.build();
+
+      DriverConfigLoader configLoader =
```

Comment:
Propagate the CL if no config loader is defined.

Reply:
This way we avoid having to specify the CL twice as we currently have to:

```
CqlSession session = CqlSession.builder()
    .withClassLoader(classLoader)
    .withConfigLoader(new DefaultDriverConfigLoader(classLoader))
    .build();
```

Reply:
sgtm 👍